lock-impl/src/main/java/com/palantir/lock/logger/LockServiceStateLogger.java

```diff
@@ -65,40 +67,31 @@ public LockServiceStateLogger(
     }
 
     public LogState logLocks() {
-        List<SimpleLockRequestsWithSameDescriptor> generatedOutstandingRequests =
-                generateOutstandingLocks(outstandingLockRequests);
-        Multimap<ObfuscatedLockDescriptor, SimpleTokenInfo> generatedHeldLocks = generateHeldLocks(heldLocks);
-        Map<ObfuscatedLockDescriptor, String> generatedSyncState = generateSyncState(descriptorToLockMap);
-        Map<ClientId, List<SanitizedLockRequestProgress>> synthesizedRequestState =
-                synthesizeRequestState(outstandingLockRequests, descriptorToLockMap);
-
         LogState state = ImmutableLogState.builder()
-                .outstandingRequests(generatedOutstandingRequests)
-                .heldLocks(generatedHeldLocks)
-                .syncState(generatedSyncState)
-                .synthesizedRequestState(synthesizedRequestState)
+                .outstandingRequests(generateOutstandingLocks(outstandingLockRequests))
+                .heldLocks(generateHeldLocks(heldLocks))
+                .syncState(generateSyncState(descriptorToLockMap))
+                .synthesizedRequestState(synthesizeRequestState(outstandingLockRequests, descriptorToLockMap))
                 .lockDescriptorMapping(lockDescriptorMapper.getReversedMapper())
                 .lockStats(lockStats)
                 .build();
         state.logTo(log);
         return state;
     }
 
-    private Map<ClientId, List<SanitizedLockRequestProgress>> synthesizeRequestState(
+    @Unsafe
+    private ListMultimap<ClientId, SanitizedLockRequestProgress> synthesizeRequestState(
```

Comment:
this should be safe still, right?

Reply:
similar to above, I don't think safety is propagating through streams & multimaps here. refactoring without streams seems to work.

```
> Task :lock-impl:compileJava
/Volumes/git/palantir/atlasdb/lock-impl/src/main/java/com/palantir/lock/logger/LockServiceStateLogger.java:87: error: [IllegalSafeLoggingArgument] Dangerous return value: result is 'UNSAFE' but the method is annotated 'SAFE'.
        return debugger.getSuspectedLockProgress().asMap().entrySet().stream()
        ^
    (see https://github.com/palantir/gradle-baseline#baseline-error-prone-checks)
```